packages/contracts-bedrock/scripts/Config.sol

```diff
@@ -94,7 +95,7 @@ library Config {
             return "optimism-goerli";
```

Comment:
In the future we should move away from using this function but fine to keep for now as it would be a breaking change, its just not easy to maintain and also results in implicit behavior that is non standard based on chain id as not all chain ids are considered "known"